### DISCUSSION TITLE
IP Address / DHCP issues

### DISCUSSION THREAD

**Discussion by idominiki** [jomjol/AI-on-the-edge-device#3194] [General] [answered]
Hey!

I just switched from AVM Fritz products to UniFi. I also changed the network to 192.168.1.0/24. 

All devices got there new IP address except ESP32. 

A few reboots doesn't helped and also assigning a static IP in UniFi doesn't work. 

In UniFi the ESP32 is shown with its old IP address. 

How can I adjust this?

Thanks 
Dominik

**Comment by caco3** [comment, marked as answer]
Are you sure the device was configured to use DHCP?
You can check the content of the `wlan.ini` file on the sd card, see https://github.com/jomjol/AI-on-the-edge-device/blob/rolling/sd-card/wlan.ini#L17-20

Note: this file is for security reasons not accessible through the Web UI, so you must take the sd card out and put it into a PC to read/edit it!

**Reply by idominiki** [reply]
Thanks, this was the solution
